pages/api/graphql-rest.page.ts

```diff
@@ -15,6 +15,18 @@ const typeDefs = gql`
     taskAnalytics(accountListId: ID!): TaskAnalytics!
   }
 
+  type Mutation {
+    exportContacts(input: ExportContactsInput!): String!
+  }
+
+  input ExportContactsInput {
+    format: String!
+    mailing: Boolean!
```

Comment:
Can you add a description? https://www.apollographql.com/docs/apollo-server/schema/schema/#documentation-strings

It's not immediately clear to me what this filter is doing. Is it only including mailing contacts when true? excluding mailing contacts when false?

A name change is another option if you like that any better